src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/AgencyInternalLocation.java

```diff
@@ -45,6 +51,14 @@ public class AgencyInternalLocation {
     @Column(name = "INTERNAL_LOCATION_TYPE")
     private String locationType;
 
+    @ManyToOne
+    @NotFound(action = IGNORE)
+    @JoinColumnsOrFormulas(value = {
+        @JoinColumnOrFormula(formula = @JoinFormula(value = "'" + LivingUnitReferenceCode.DOMAIN + "'", referencedColumnName = "domain")),
+        @JoinColumnOrFormula(column = @JoinColumn(name = "INTERNAL_LOCATION_TYPE", referencedColumnName = "code", insertable = false, updatable = false))
```

Comment:
didn't want to change all existing code esp. api responses so have made this not insertable or updateable, so field is effectively duplicated